frontend/src/containers/DonatePage.js

```diff
@@ -238,6 +231,6 @@ function mapStateToProps({
     saveInProgress: users.updateInProgress,
     isAuthenticated: session.isAuthenticated,
     newUser,
-    i18n
+    i18n: i18n(group.settings.lang)
```

Comment:
You are absolutely right. This whole thing is a hack because we need something out the door quickly. We should use a i18n library and save the `lang` as a setting of the group.